inscriptions/ISic000707.xml

```diff
@@ -191,11 +196,16 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AL">
+                <ab>
+                    <lb n="1"/><roleName><supplied reason="lost"><expan><abbr>Imp</abbr><ex>erator</ex></expan></supplied></roleName> <roleName><supplied reason="lost"><expan><abbr>Caes</abbr><ex>ar</ex></expan></supplied></roleName> <persName type="emperor"><name type="praenomen"><supplied reason="lost">M</supplied>arcu<supplied reason="lost">s</supplied></name> <name type="nomen"><supplied reason="lost">Aurelius</supplied></name> <name type="cognomen"><supplied reason="lost">Antoninus</supplied></name>
```

Comment:
the rolenames would fall inside the persName, since titles of the named individual. For current thinking / guidance, see https://github.com/ISicily/ISicily/wiki/Encoding:-text (where we suggest that on the persName here you would give type="attested" and role="emperor")